lib/perl5/FlyBase/Proforma/Util.pm

```diff
@@ -5069,7 +5069,8 @@ sub check_al_with_fr_or_mutagen {
 		where feature.uniquename='$fb_id' and
 		feature.feature_id=fcv.feature_id and
 		cvterm.cvterm_id=fcv.cvterm_id and
-		cvterm.name like 'in vitro construct%';";
+        cvterm.is_obsolete = 0 and
+		cvterm.name = 'in vitro construct';";
```

Comment:
this looks like it is OK to me, but as my sql is not so good, I think you should also get approval from Ian for this one

Reply:
okay by me. rerunning the tests now after updating the tested and test suite.